### ISSUE TITLE
They removed the bootstrap asset

### ISSUE THREAD

**Issue by corentinchap** [xjflyttp/yii2-notifyjs-widget#1]
NotifyJS removed the bootstrap asset, doesn't appear in bower/notifyjs/dist/styles
Only metro is available
https://github.com/jpillora/notifyjs/tree/master/dist

**Comment by xjflyttp** [comment]
Yes, bootstrap theme include main js, you can direct use it. I will remove bootstrap asset later.
Thank you.
see source
```
https://github.com/jpillora/notifyjs/blob/0b8891e/dist/notify.js#L531-L571
```
